### PR TITLE
fix translation in compare validator

### PR DESCRIPTION
**Pull request by ZAYEC77**

### CODE REVIEW

**Comment by samdark** [comment]
What was exactly wrong with current one?

**Comment by SilverFire** [comment]
I think it refers to https://github.com/yiisoft/yii2/pull/7632, but in a wrong direction.

@ZAYEC77, `compareValue` was changed to `compareValueOrAttribute` not by an accident.
If you want to help with translations - I would kindly ask you to translate those messages with `compareValueOrAttribute` to RU and UA.

**Comment by ZAYEC77** [comment]
All translations with key `compareValue` were present in translations files. 
Do You propose to change translations (more than 10 items) in majority of languages?

**Comment by SilverFire** [comment]
Yes, I do. These messages were changed in the `CompareValidator` and have to be updated in all langs

**Comment by ZAYEC77** [comment]
@SilverFire okay, thanks and what about method `clientValidateAttribute`. It has lines:
```php
$options['message'] = Yii::$app->getI18n()->format($this->message, [
            'attribute' => $model->getAttributeLabel($attribute),
            'compareAttribute' => $compareValue,
            'compareValue' => $compareValue,
        ], Yii::$app->language);
```

maybe is better to change translation key there?

**Comment by SilverFire** [comment]
@ZAYEC77 oh yes, it was missed. Thank you for attention.
The same problem is in `validateValue()` method. Can you fix it, please?

**Comment by ZAYEC77** [comment]
@SilverFire yeap, I will try to do it

File: framework/messages/ru/yii.php [hunk 1]
```diff
@@ -17,28 +17,12 @@
...
-    '{nFormatted} {n, plural, =1{mebibyte} other{mebibytes}}' => '{nFormatted} {n, plural, one{мебибайт} few{мебибайта} many{мебибайтов} other{мебибайта}}',
-    '{nFormatted} {n, plural, =1{megabyte} other{megabytes}}' => '{nFormatted} {n, plural, one{мегабайт} few{мегабайта} many{мегабайтов} other{мегабайта}}',
-    '{nFormatted} {n, plural, =1{pebibyte} other{pebibytes}}' => '{nFormatted} {n, plural, one{пебибайт} few{пебибайта} many{пебибайтов} other{пебибайта}}',
-    '{nFormatted} {n, plural, =1{petabyte} other{petabytes}}' => '{nFormatted} {n, plural, one{петабайт} few{петабайта} many{петабайтов} other{петабайта}}',
-    '{nFormatted} {n, plural, =1{tebibyte} other{tebibytes}}' => '{nFormatted} {n, plural, one{тебибайт} few{тебибайта} many{тебибайтов} other{тебибайта}}',
-    '{nFormatted} {n, plural, =1{terabyte} other{terabytes}}' => '{nFormatted} {n, plural, one{терабайт} few{терабайта} many{терабайтов} other{терабайта}}',
+    '{attribute} must be equal to "{compareValueOrAttribute}".' => 'Значение «{attribute}» должно быть равно «{compareValue}».',
```
Comment (SilverFire) [inline framework/messages/ru/yii.php]: На русском осталось `compareValue`, нужно тоже `compareValueOrAttribute`

**Comment by SilverFire** [comment]
Merged https://github.com/yiisoft/yii2/commit/e6e8b0a41f313ce42c62a71e462cb3b0fd87ccdd

**Comment by SilverFire** [comment]
Thank you!

**Pull request closed without merging**